docs_src/microservices/device/supported/device-usb-camera/supplementary-info/discovery.md

```diff
@@ -1,5 +1,5 @@
 # Dynamic Discovery
-The device service supports [dynamic discovery](https://docs.edgexfoundry.org/2.1/microservices/device/Ch-DeviceServices/#dynamic-provisioning).
+The device service supports [dynamic discovery](https://docs.edgexfoundry.org/{{version}}/microservices/device/Ch-DeviceServices/#dynamic-provisioning).
```

Comment:
This should a relative link since it part of the docs.

```suggestion
The device service supports [dynamic discovery](../../../../Ch-DeviceServices/#dynamic-provisioning).
```